src/Php80/Php80.php

```diff
@@ -26,6 +26,11 @@ public static function fdiv($dividend, $divisor)
         return (float) @($dividend / $divisor);
     }
 
+    public static function str_contains($haystack, $needle)
```

Comment:
please add all type declarations

Reply:
Are you sure? We are not using them in other functions yet.

Reply:
See #234

Reply:
(please rebase again while doing so)

Reply:
OK, typehints added, changes rebased again 😉